avalon/tools/cbloader/delegates.py

```diff
@@ -106,7 +106,9 @@ def displayText(self, value, locale):
 
 class VersionDelegate(QtWidgets.QStyledItemDelegate):
     """A delegate that display version integer formatted as version string.
-    
+
+    Note:
+        It is required to set source_model attribute.
```

Comment:
This added docstring note isn't actually true so can be removed (that was only from some debugging in development)